client/me/two-step/index.jsx

```diff
@@ -1,49 +1,56 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:me:two-step' );
+import debugFactory from 'debug';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+const debug = debugFactory( 'calypso:me:two-step' );
```

Comment:
Should the `const debug` go after all imports? That's the convention I've seen.

Reply:
Right you are - let me tweak this.